test/cases/generation.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/bash
+set -euxo pipefail
+
+# Provision the software under test.
+/usr/libexec/osbuild-composer-test/provision.sh
+
+# Get OS data
+source /usr/libexec/osbuild-composer-test/set-env-variables.sh
+
+# Colorful timestamped output.
+function greenprint {
+    echo -e "\033[1;32m[$(date -Isecond)] ${1}\033[0m"
+}
+
+# install requirements
+sudo dnf -q -y install make \
+    git \
+    go \
+    ranger \
+    openssl \
+    rpm-build \
+    aws \
+    qemu \
+    setools-console \
+    setroubleshoot \
+    container-selinux \
+    lvm2
+
+# stop dnf-json socket
+sudo systemctl stop osbuild-dnf-json.socket
+
+# test the test cases generation
+WORKDIR=$(mktemp -d)
+mkdir -p "$WORKDIR"
```

Comment:
This creates a temporary directory in `/tmp`, which is on `tmpfs` and based on the logs from the CI, there is not enough space for osbuild to store all of its data in the store directory. We usually use `/var/tmp`, which is on the disk and not in the RAM. I think that should solve the CI failure.